docs/ballerina-to-oas/spec/spec.md

```diff
@@ -1214,6 +1216,51 @@ oneOf:
 >       type: string
 >   additionalProperties: false
 > ```
+> 2. If we have a record defined in a separate package, for example, packageA:
+> ```ballerina
+> type RecA record {|
+>    int a = 10;
+>    string c = "c";
+>|};
+>
+> ```
+> and it is included in packageB:
+>```ballerina
+> type RecB record {|
+>    *packageA:RecA;
+>     string b = "c";
+>|};
+>```
+> In the OpenAPI Specification (OAS) mapping for `RecB`, the default value access API cannot retrieve default values
+> from another package. To address this, we generate all the fields explicitly for `RecB` to ensure accessibility.
+> ```yaml
+> RecA:
+>   type: object
+>   properties:
+>     a:
+>       type: integer
+>       default: 10
```

Comment:
We can't get the default value for the external package right?

Reply:
Yeah ,will update the example